server/templates/station.html

```diff
@@ -15,6 +15,11 @@ <h3>Ground Station  {{ station.name }}</h3>
                     [List]
                 </a>
             </li>
+            <li>Station admins:
+            {% for o in owners %}
+                <span class="param"">{{o.username}} (id: {{o.id}})</span>
```

Comment:
The user probably shouldn't see our internal ID.

Reply:
Ok, removed.